File: PRAKTIKUM-PEMROGRAMAN1/MODUL3/Python/PRAK304-2410817210024-MuhammadIbnuSina.py
bilangan= int(input())

if bilangan == 0:
    print("Nol")
elif bilangan >= 1 and bilangan < 10:
    print("Satuan")
elif bilangan >10 and bilangan < 20:
    print("Belasan")
elif bilangan == 10 or bilangan > 19 and bilangan < 100:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL3/C/PRAK304-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

int main(){
    int bilangan;
    scanf("%d", &bilangan);

    if (bilangan == 0){
        printf("Nol");
    }else if (bilangan >= 1 && bilangan < 10){
        printf("Satuan");
    }else if (bilangan >10 && bilangan < 20){
        printf("Belasan");
    }else if (bilangan == 10 || bilangan > 19 && bilangan < 100){
        printf("Puluhan");
    }else{
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}